CMakeLists.txt

```diff
@@ -945,17 +946,19 @@ if(BUILD_NVFUSER_BENCHMARK)
       ${CMAKE_SOURCE_DIR}/third_party/benchmark/include
       ${CMAKE_SOURCE_DIR}/third_party/flatbuffers/include
       ${CMAKE_SOURCE_DIR}/third_party/googletest/googletest/include
+      ${CMAKE_SOURCE_DIR}/third_party/googletest/googlemock/include
     )
     target_include_directories(nvfuser_multidevice_bench PUBLIC ${NVFUSER_ROOT})
     target_link_libraries(nvfuser_multidevice_bench PRIVATE
       GTest::gtest
+      GTest::gmock
       benchmark::benchmark
       codegen_internal
     )
     add_dependencies(nvfuser_multidevice_bench flatc build_flatbuffer_config)
 
     if(NOT MSVC)
-      target_compile_options(nvfuser_bench PRIVATE
+      target_compile_options(nvfuser_multidevice_bench PRIVATE
```

Comment:
ditto

Reply:
this is unrelated, but a bug IIUC. I revert it here to merge the current PR asap.